python/py-absl/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-absl
```

Comment:
please add `python.rootname  absl-py` here and then remove the `master_sites`, `distname`, and `livecheck.name` fields below